src/viewer.js

```diff
@@ -1527,6 +1562,32 @@ class VolumeImageViewer {
     })
   }
 
+  /**
+   * Updates the style of a feature.
+   *
+   * @param {Feature} feature - Feature
+   * @param {object} styleOptions - Style options
+   * @param {object} styleOptions.stroke - Style options for the outline of the geometry
+   * @param {number[]} styleOptions.stroke.color - RGBA color of the outline
+   * @param {number} styleOptions.stroke.width - Width of the outline
+   * @param {object} styleOptions.fill - Style options for body the geometry
+   * @param {number[]} styleOptions.fill.color - RGBA color of the body
+   * @param {object} styleOptions.image - Style options for image
+   */
+  setFeatureStyle (feature, styleOptions, optSilent = false) {
```

Comment:
The public API of the viewer should not expose any of the underlying OpenLayers types. We will need to be abstract this.